cmd/tests/cmd_run_test.go

```diff
@@ -2158,6 +2159,13 @@ func BenchmarkRunEvents(b *testing.B) {
 func TestBrowserPermissions(t *testing.T) {
 	t.Parallel()
 
+	// This will skip running this test if chrome or chromium is not installed
+	// which is required to run the test to ensure that the browser module test
+	// only runs when the prerequisite browser options are supplied.
+	if !foundChrome(t) {
+		t.Skip()
+	}
+
```

Comment:
Two problems with this:
1. if we stop having chrome in any CI it will stop actually runnign this test at all
2. this still runs chromium for this test.

I would kind of recommend trying to run a command that *won't* work and then expect that:
1. if no borwser optiosn - we get one error
2. if browser option is available - we get the error that you can't run the "fake" command we chose.

Reply:
> if browser option is available - we get the error that you can't run the "fake" command we chose.
That's a good option. I'll give that a go.

Reply:
Resolved in 04f2fb9baee3411003704aea80cbccc70e46600d